src/us/kbase/workspace/test/workspace/WorkspaceUnitTest.java

```diff
@@ -65,16 +67,20 @@ private TestMocks initMocks() {
 		@SuppressWarnings("unused")
 		private final ResourceUsageConfiguration cfg;
 		private final Workspace ws;
+		@SuppressWarnings("unused")
+		private TempFilesManager tfm;
```

Comment:
this actually is used to create the workspace, so I'd prefer to include it in the TestMocks class so the next person doesn't have to dig around if they need to use the tfm mock